x-pack/plugins/cases/server/client/utils.test.ts

```diff
@@ -906,7 +906,7 @@ describe('utils', () => {
           ...customFieldsConfiguration,
           {
             key: 'fourth_key',
-            type: 'number',
+            type: 'symbol',
```

Comment:
Why `'symbol'` here?

Reply:
Yes. In type property we need to have not existing for cases customFields type. It was number previously. But now I introduced number and test failed. So I've change it to something which does not exist as a type. Like symbol. Do you have another suggestion?

Reply:
`foobar`? :D but it's not important, just wanted to understand what happened ;)